archive/frames/views.py

```diff
@@ -208,27 +208,37 @@ def aggregate(self, request):
                 status=status.HTTP_400_BAD_REQUEST
```

Comment:
Can you remove these changes? Let's scope this PR only to the count optimization.

Reply:
Is this present in another branch? Or are we wanting to test these in isolation?

I don't necessarily see the need to remove these unless that's what we're going after.